NSS-Day-Cohort-66/rare-api-eager-miracles
# Integrations Test for PostReactions

## Overview:

- Made tests for list and create methods on PostReactions

## Testing:

1. Git fetch and checkout this branch `lh-postreactions-tests` in parent API directory
2. Run ` python3 manage.py test tests -v 1 ` in terminal
3. Verify success message in terminal:
```
Ran 21 tests in 0.100s

OK
Destroying test database for alias 'default'...
```
